<ui version="4.0" >
 <class>STControllerMainWindow</class>
 <widget class="QMainWindow" name="STControllerMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SequenceTree4 Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>33</y>
     <width>569</width>
     <height>548</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>545</width>
         <height>503</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Activity Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QTextEdit" name="log_text" >
          <property name="lineWrapMode" >
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>545</width>
         <height>503</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Sequences</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QFrame" name="frame_2" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4" >
           <item>
            <widget class="QPushButton" name="refresh_sequences_button" >
             <property name="text" >
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="open_sequence_folder_button" >
             <property name="text" >
              <string>Open Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>423</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="SequencesTree" name="sequences" >
          <column>
           <property name="text" >
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>545</width>
         <height>503</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Raw Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QFrame" name="frame" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" >
           <item>
            <widget class="QPushButton" name="refresh_raw_data_button" >
             <property name="text" >
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="view_header_button" >
             <property name="text" >
              <string>View Header</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="parse_button" >
             <property name="text" >
              <string>Parse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="open_raw_folder_button" >
             <property name="text" >
              <string>Open Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>261</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="RawDataFileTree" name="raw_data_files" >
          <column>
           <property name="text" >
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>545</width>
         <height>503</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Experiments</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionConfiguration" />
   </widget>
   <addaction name="menuTools" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>581</y>
     <width>569</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>569</width>
     <height>12</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConfiguration" >
   <property name="text" >
    <string>Controller Configuration...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RawDataFileTree</class>
   <extends>QTreeWidget</extends>
   <header>rawdatafiletree.h</header>
  </customwidget>
  <customwidget>
   <class>SequencesTree</class>
   <extends>QTreeWidget</extends>
   <header>sequencestree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
